automation/build-bin.ts

```diff
@@ -253,7 +253,7 @@ async function signWindowsInstaller() {
 			'-sha1',
 			process.env.SM_CODE_SIGNING_CERT_SHA1_HASH,
 			'-tr',
-			process.env.TIMESTAMP_SERVER || 'http://timestamp.comodoca.com',
+			process.env.TIMESTAMP_SERVER ?? 'http://timestamp.comodoca.com',
```

Comment:
I think `process.env.X` can be `''` as well in which case we want to use `||` for all of them, not `??`